src/components/name-output/name-output.component.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef, useState } from "react";
+import EditName from "../name/name.component";
+import "./name-output.styles.css";
+
+const firebase = require("firebase");
```

Comment:
This is not needed. We have imported this from index.firebase.js

Reply:
Well noted.